suite-native/accounts/src/components/AccountListPlaceholder.tsx

```diff
@@ -1,24 +1,32 @@
 import { Box, Pictogram } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useTranslate } from '@suite-native/intl';
 
 const PLACEHOLDER_HEIGHT = 380;
 
+type AccountListPlaceholderProps = {
+    isFilterEmpty?: boolean;
+};
+
 const titleVariant = prepareNativeStyle(_ => ({
     justifyContent: 'center',
     alignItems: 'center',
     height: PLACEHOLDER_HEIGHT,
 }));
 
-export const AccountListPlaceholder = () => {
+export const AccountListPlaceholder = ({ isFilterEmpty }: AccountListPlaceholderProps) => {
+    const { translate } = useTranslate();
     const { applyStyle } = useNativeStyles();
 
     return (
         <Box style={applyStyle(titleVariant)}>
             <Pictogram
                 variant="yellow"
                 icon="searchLight"
```

Comment:
The icon does not match. `searchLight` should be displayed only if the user is searching. On empty device should be displayed coins icon.